cli/command/stack/common.go

```diff
@@ -36,7 +42,7 @@ func getServices(
 ) ([]swarm.Service, error) {
 	return apiclient.ServiceList(
 		ctx,
-		types.ServiceListOptions{Filters: getStackFilter(namespace)})
+		types.ServiceListOptions{Filters: getServiceFilter(namespace)})
```

Comment:
(same comment as above )

These are intended to be used internally for filtering. Should we doc these now or wait until we support the custom filtering based upon runtime type?